addons/im_livechat/static/src/public_models/livechat_button_view.js

```diff
@@ -209,10 +209,13 @@ Model({
         /**
          * @private
          */
-        _openChat() {
+        async _openChat() {
             if (this.isOpeningChat) {
                 return;
             }
+            if (!this.env.services["bus_service"].isWorkerStarted) {
+                await this.env.services["bus_service"].startWorker();
```

Comment:
I don't think it should be the called responsibility to start the bus worker. Indeed, one could easily call `busService.start/busService.addChannel` and forget to properly start the underlying worker. I'd rather having those two methods start the worker if necessary when called.

Reply:
done (start & addChannel start the worker, open chat triggers addChannel)